### ISSUE TITLE
The buttons that go in the lower left part of the config only show up in the local overrides menu

### ISSUE THREAD

**Issue by dombomb64** [enjarai/show-me-your-skin#91]
The buttons with the sword, name tag, and elytra only show up when editing the client-side overrides. I don’t know if this is a bug, but I feel like the buttons should stay for the global settings menu. If I’m understanding correctly, the local overrides only show up locally, and the fact that the three buttons disappear when editing yourself for other players makes me think that you can’t disable the combat setting, your elytra, or your name tag without telling everybody else to do the same in their local overrides as well.

TL;DR, Can the combat, name tag, and elytra buttons be in global settings as well?

**Comment by enjarai** [comment]
The three buttons you are refferring to can be exploited in for example PVP situations, as such, they're locked behind a server-side config option. You or the owner of the server you play on has to modify the `config/show-me-your-skin-server.json` file to allow these settings to be used globally.
